<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>MnemonicWidget</class>
<widget class="QDialog" name="MnemonicWidget">
 <property name="geometry">
  <rect>
   <x>0</x>
   <y>0</y>
   <width>841</width>
   <height>643</height>
  </rect>
 </property>
 <property name="windowTitle">
  <string>Form</string>
 </property>
 <property name="styleSheet">
  <string notr="true">border:none;
   padding:0px;
   margin:0px;</string>
 </property>
 <layout class="QHBoxLayout" name="horizontalLayout_2">
  <property name="spacing">
   <number>0</number>
  </property>
  <property name="leftMargin">
   <number>0</number>
  </property>
  <property name="topMargin">
   <number>0</number>
  </property>
  <property name="rightMargin">
   <number>0</number>
  </property>
  <property name="bottomMargin">
   <number>0</number>
  </property>
  <item>
   <widget class="QFrame" name="containerLeft">
    <property name="styleSheet">
     <string notr="true">
      background-color:#161616;</string>
    </property>    
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#containerLeft {
      border-image: url(&quot;:/icons/Banner_inicio2&quot;) 0 0 0 0 stretch stretch;
      padding:0px;
      margin:0px;
      border:none;
      }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="imgTutorial">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="messageTop">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:20px;
         font-weight:bold;
         color:#FFFFFF;</string>
       </property>
       <property name="text">
        <string>Backup your recovery seed phrase</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="messageBottom">
       <property name="styleSheet">
        <string notr="true">font-size:16px;
         color:#FFFFFF;
         padding:0 20px;</string>
       </property>
       <property name="text">
        <string>Your 24-word seed phrase can
         be used to restore your wallet.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </item>
  <item>
   <widget class="QFrame" name="frame_2">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left:16px;
      padding-right:16px;
      background-color:#161616;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="QStackTutorialContainer">
       <property name="styleSheet">
        <string notr="true">#frame2{
         background-color: #161616;
         border: none;
         padding: 0em 0em;
         }</string>
       </property>
       <widget class="QWidget" name="page_3"/>
       <widget class="QWidget" name="page_4"/>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="line_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>1</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#161616;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="containerBtns" stretch="1,0,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnBack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">#btnBack {
           color:#105aef;
           border: none;
           padding: 0em 0em;
           }

           #btnBack:hover {
           color:#0e45b4;
           }</string>
         </property>
         <property name="text">
          <string>BACK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="btnLineSeparator">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1</width>
           <height>36</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#161616;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNext">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">#btnNext {
           color:#105aef;
           border: none;
           padding: 0em 0em;
           }

           #btnNext:focus {
           background-color: rgb(255, 255, 255);
           border:none;
           }

           #btnNext:hover {
           color:#0e45b4;
           }

           #btnNext:checked{
           background-color: none;
           border: none;
           }</string>
         </property>
         <property name="text">
          <string>NEXT</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </item>
 </layout>
</widget>
<resources/>
<connections/>
</ui>
